### PR TITLE
Issue #3335 - fix EEModuleDescriptor warnings

### LINKED ISSUE #3335: CDI: EEModuleDescriptor warnings from weld
Warnings such as this one have started showing up since we moved up to the latest versions of Weld

W WELD-000718: No EEModuleDescriptor defined for bean archive with ID: TestVetoedAlternative#TestVetoedAlternative.war. @Initialized and @Destroyed events for ApplicationScoped may be fired twice.

### PR DESCRIPTION
**Pull request by tevans78**
fixes #3335

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/WebSphereEEModuleDescriptor.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import com.ibm.ws.cdi.internal.interfaces.ArchiveType;
+
+/**
+ *
+ */
+public class WebSphereEEModuleDescriptor extends EEModuleDescriptorImpl {
+
+    /**
+     * @param id
+     * @param moduleType
```
Comment (hutchig) [inline dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/WebSphereEEModuleDescriptor.java]: It is not a moduleType but an archiveType

**Comment by tevans78** [comment]
One unrelated faultolerance failure in personal build. merging.

**Pull request merged**